### ISSUE TITLE
Spacing/Visual

### ISSUE THREAD

**Issue by CloVasco** [benetech/ServiceNet#388]
Need to improve the visual of the platform, particularly the spacing.
Part of the big overhaul and rethinking of the platform look and feel and navigation

**Comment by NeilMBenetech** [comment]
Yes all the fields should line up side-by-side. Things like a longer organizational name at the very top that wraps to second line are pushing down the rest of the column, causing fields to misalign and making it more difficult to do the comparison. This is happening in either column.

Example, logged in as "hluser" and viewing this record: https://servicenet.benetech.org/#/multi-record-view/dd602036-4298-4d4d-a1e7-2f3c87b06175
